hudi-common/src/test/java/org/apache/hudi/common/testutils/reader/HoodieFileSliceTestUtils.java

```diff
@@ -207,7 +208,7 @@ private static HoodieDataBlock createDataBlock(
             false,
             header,
             HoodieRecord.RECORD_KEY_METADATA_FIELD,
-            CompressionCodecName.GZIP,
+            "gzip",
```

Comment:
No way to put this name into the enum?

Reply:
Compression.Algorithm.GZ.name? or something?

Reply:
Replaced such occurrences with the default config value.